core/iwasm/interpreter/wasm_interp_classic.c

```diff
@@ -3879,6 +4444,65 @@ wasm_interp_call_func_bytecode(WASMModuleInstance *module,
             if (cur_func->import_func_inst) {
                 wasm_interp_call_func_import(module, exec_env, cur_func,
                                              prev_frame);
+#if WASM_ENABLE_EXCE_HANDLING != 0
+                const char *uncaught_exception = wasm_get_exception(module);
+                if (uncaught_exception
+                    && strcmp(uncaught_exception,
+                              "Exception: uncaught wasm exception")
+                           == 0) {
```

Comment:
Maybe it is better to use `if (has_exception && strstr(uncaught_exception, "uncaught wasm exception")`?

Reply:
updated according to recommendation.

Reply:
Updated accordingly